reserved/persona.go

```diff
@@ -0,0 +1,51 @@
+package reserved
+
+import (
+	"log"
+	"math"
+
+	"github.com/wopta/goworkspace/models"
+)
+
+func personaReserved(policy *models.Policy) (bool, *models.ReservedInfo) {
+	log.Println("[personaReserved]")
+
+	reservedInfo := &models.ReservedInfo{
+		Reasons:       make([]string, 0),
+		RequiredExams: make([]string, 0),
+	}
+
+	isReserved := false
+	reason := "BMI index out of range"
```

Comment:
Change it into a constant so we know that it's value can't/shouldn't be changed. For now it can be inside this function as it's not used elsewhere.